osf/models/metaschema.py

```diff
@@ -148,6 +149,43 @@ def validate_metadata(self, metadata, reviewer=False, required_fields=False):
             raise ValidationValueError(e.message)
         return
 
+    def validate_registration_responses(self, registration_responses, required_fields=False):
+        """
+        Validates registration_responses against the cached jsonschema on the RegistrationSchema.
+        The `title` of the question is stashed under the description for the particular question property
+        for forumulating a more clear error response.
+        """
+        validation_schema = self.registration_responses_jsonschema
+        if not required_fields and validation_schema.get('required', None):
+            del validation_schema['required']
+
+        try:
+            jsonschema.validate(registration_responses, validation_schema)
+        except jsonschema.ValidationError as e:
+            properties = validation_schema.get('properties', {})
+            relative_path = getattr(e, 'relative_path', None)
+            question_id = relative_path[0] if relative_path else ''
+            if properties.get(question_id, None):
+                question_title = properties.get(question_id).get('description') or question_id
```

Comment:
Annotated the question title, where it existed on the `description` field on the jsonschema, so it could be used here in making a clearer error message.  Input welcome on better jsonschema fields to annotate with.  Looked like `title` was only top-level.